### ISSUE TITLE
bitbucket Cloud Support

### ISSUE THREAD

**Issue by kappy** [mendhak/teamcity-stash#27]
Is this able to also push the status to the Bitbucket Cloud using the api?

http://blog.bitbucket.org/2015/11/18/introducing-the-build-status-api-for-bitbucket-cloud/

**Comment by mendhak** [comment]
I don't believe this will work - the Bitbucket Cloud API is quite new and I don't know if Bitbucket Server (formerly stash) has that API locally.  

JetBrains do have a general purpose commit publisher though and they seem to work with Bitbucket Cloud:  https://github.com/JetBrains/commit-status-publisher

**Comment by kappy** [comment]
Indeed. After posting the issue, I was reading the code and the API that this library uses to submit the build status is not compatible with Bitbucket Cloud.

I did tried the Commit-Status-Publisher, but didn't got it working after installing the plugin, so I started to look for alternatives and found your repository. (Kudos btw :smile:  )

I guess I'll need to put a little bit more effort trying to put the other one working. 
Thanks :+1:
